src/Microsoft.Diagnostics.DebugServices/RuntimeEnumerationFlags.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Diagnostics.DebugServices
+{
+    /// <summary>
+    /// Enumeration control flags
+    /// </summary>
+    [Flags]
+    public enum RuntimeEnumerationFlags
+    {
+        /// <summary>
+        /// Providers can return only the runtimes they feel are important like ones involved in a crash.
+        /// </summary>
+        Default = 0x00,
```

Comment:
This flag is used to specify that don't want all of the runtimes enumerated. How do I specify which ones to be enumerated?

Reply:
Currently there are only 2 choices: the default which tries to find the crashing runtime (via the SpecialDiagInfo memory block) or "All" which goes through all the modules in the target process to find runtimes.